hoomd/hpmc/pytest/test_clusters.py

```diff
@@ -155,57 +137,10 @@ def test_valid_setattr_attached(attr, value, simulation_factory,
     setattr(cl, attr, value)
     assert getattr(cl, attr) == value
 
-
-@pytest.mark.parametrize("delta_mu", [-2.0, 0, 2.0])
-def test_swap_moves(delta_mu, simulation_factory,
-                    lattice_snapshot_factory):
-    """Test that Clusters can swap types."""
-
-    sim = simulation_factory(lattice_snapshot_factory(particle_types=['A', 'B'],
-                                                      dimensions=3, a=4, n=7, r=0.1))
-
-
-    mc = hoomd.hpmc.integrate.Sphere(seed=1, d=0.1, a=0.1)
-    mc.shape['A'] = dict(diameter=1)
-    mc.shape['B'] = dict(diameter=1)
-    sim.operations.integrator = mc
-
-    cl = hoomd.hpmc.update.Clusters(trigger=hoomd.trigger.Periodic(5),
-                                    swap_type_pair=['A', 'B'],
-                                    swap_move_ratio=1,
-                                    delta_mu=delta_mu,
-                                    seed=12)
-    sim.operations.updaters.append(cl)
-
-    # set every other particle to type B (typeid=1)
-    snap = sim.state.snapshot
-    if snap.exists:
-        snap.particles.typeid[range(0, sim.state.N_particles, 2)] = 1
-    sim.state.snapshot = snap
-
-    # number of particles of each type should change after a run
-    sim.run(100)
-
-    snap = sim.state.snapshot
-    acceptance = cl.swap_moves[0]/np.sum(cl.swap_moves)
-    if snap.exists:
-        num_type_B = np.sum(snap.particles.typeid)
-        num_type_A = len(snap.particles.typeid) - num_type_B
-        if delta_mu<0:
-            assert num_type_B < num_type_A
-            assert acceptance < 1
-        elif delta_mu == 0.0:
-            # ALL swap moves should be accepted when delta_mu = 0
-            assert np.isclose(acceptance, 1.0)
-        elif delta_mu>0:
-            assert num_type_A < num_type_B
-            assert acceptance < 1
-
-
-@pytest.mark.parametrize("delta_mu", [-2.0, 0, 2.0])
-def test_pivot_moves(delta_mu, simulation_factory,
+@pytest.mark.serial
```

Comment:
Does `Cluster` no longer work in MPI simulations?

Reply:
No, it won't work. I removed the domain decomposition support (which was complicated and inefficient anyway). Cluster moves are global, so they don't map well onto a spatial domain decomposition.